core/capabilities/syncer.go

```diff
@@ -241,6 +195,101 @@ func (s *registrySyncer) Name() string {
+	workflowDonPeers  []string
+	triggerDonPeers   []string
+	triggerDonSigners []string
+
+	workflowsDonInfo         capabilities.DON
+	triggerCapabilityDonInfo capabilities.DON
+}
+
+func NewHardcodedDonNetworkSetup(peerWrapper p2ptypes.PeerWrapper) (HardcodedDonNetworkSetup, error) {
+	result := HardcodedDonNetworkSetup{}
+
+	result.workflowDonPeers = []string{
+		"12D3KooWBCF1XT5Wi8FzfgNCqRL76Swv8TRU3TiD4QiJm8NMNX7N",
+		"12D3KooWG1AyvwmCpZ93J8pBQUE1SuzrjDXnT4BeouncHR3jWLCG",
+		"12D3KooWGeUKZBRMbx27FUTgBwZa9Ap9Ym92mywwpuqkEtz8XWyv",
+		"12D3KooW9zYWQv3STmDeNDidyzxsJSTxoCTLicafgfeEz9nhwhC4",
+	}
+	result.triggerDonPeers = []string{
+		"12D3KooWBaiTbbRwwt2fbNifiL7Ew9tn3vds9AJE3Nf3eaVBX36m",
+		"12D3KooWS7JSY9fzSfWgbCE1S3W2LNY6ZVpRuun74moVBkKj6utE",
+		"12D3KooWMMTDXcWhpVnwrdAer1jnVARTmnr3RyT3v7Djg8ZuoBh9",
+		"12D3KooWGzVXsKxXsF4zLgxSDM8Gzx1ywq2pZef4PrHMKuVg4K3P",
+		"12D3KooWSyjmmzjVtCzwN7bXzZQFmWiJRuVcKBerNjVgL7HdLJBW",
+		"12D3KooWLGz9gzhrNsvyM6XnXS3JRkZoQdEzuAvysovnSChNK5ZK",
+		"12D3KooWAvZnvknFAfSiUYjATyhzEJLTeKvAzpcLELHi4ogM3GET",
+	}
+	result.triggerDonSigners = []string{
+		"0x9CcE7293a4Cc2621b61193135A95928735e4795F",
+		"0x3c775F20bCB2108C1A818741Ce332Bb5fe0dB925",
+		"0x50314239e2CF05555ceeD53E7F47eB2A8Eab0dbB",
+		"0xd76A4f98898c3b9A72b244476d7337b50D54BCd8",
+		"0x656A873f6895b8a03Fb112dE927d43FA54B2c92A",
+		"0x5d1e87d87bF2e0cD4Ea64F381a2dbF45e5f0a553",
+		"0x91d9b0062265514f012Eb8fABA59372fD9520f56",
+	}
+
+	allPeers := make(map[ragetypes.PeerID]p2ptypes.StreamConfig)
+	addPeersToDONInfo := func(peers []string, donInfo *capabilities.DON) error {
+		for _, peerID := range peers {
+			var p ragetypes.PeerID
+			err := p.UnmarshalText([]byte(peerID))
+			if err != nil {
+				return err
+			}
+			allPeers[p] = defaultStreamConfig
+			donInfo.Members = append(donInfo.Members, p)
+		}
+		return nil
+	}
+	result.workflowsDonInfo = capabilities.DON{ID: "workflowDon1", F: 1}
+	if err := addPeersToDONInfo(result.workflowDonPeers, &result.workflowsDonInfo); err != nil {
+		return HardcodedDonNetworkSetup{}, fmt.Errorf("failed to add peers to workflow DON info: %w", err)
+	}
+	result.triggerCapabilityDonInfo = capabilities.DON{ID: "capabilityDon1", F: 1} // NOTE: misconfiguration - should be 2
+	if err := addPeersToDONInfo(result.triggerDonPeers, &result.triggerCapabilityDonInfo); err != nil {
+		return HardcodedDonNetworkSetup{}, fmt.Errorf("failed to add peers to trigger DON info: %w", err)
+	}
+	err := peerWrapper.GetPeer().UpdateConnections(allPeers)
+	if err != nil {
+		return HardcodedDonNetworkSetup{}, fmt.Errorf("failed to update connections: %w", err)
+	}
+
+	return result, nil
+}
+
+func (h HardcodedDonNetworkSetup) IsWorkflowDon(id p2ptypes.PeerID) bool {
+	return slices.Contains(h.workflowDonPeers, id.String())
+}
+
+func (h HardcodedDonNetworkSetup) IsTriggerDon(id p2ptypes.PeerID) bool {
+	return slices.Contains(h.triggerDonPeers, id.String())
+}
+
+func (h HardcodedDonNetworkSetup) TriggerDonSigners() []string {
```

Comment:
@ettec Nit/style: why not just make the underlying attribute public rather than having a getter? Unless I'm missing something, the []string would still be mutable